packages/fs-router-vite-plugin/src/build-route-config.ts

```diff
@@ -3,7 +3,7 @@ import type { RouteNode } from './types'
 
 export function buildRouteConfig(nodes: Array<RouteNode>, depth = 1): string {
   const children = nodes.map((node) => {
-    const route = `${node.variableName}Route`
+    const route = `${node.variableName as string}Route`
```

Comment:
Why we need to cast the `string` here?

Reply:
https://github.com/tuono-labs/tuono/blob/de74647c434a0188951f7615b0e12f457a435186/packages/fs-router-vite-plugin/src/types.ts#L10

Because `variableName` is `string | undefined`

Reply:
Makes sense